tasks/Qc.wdl

```diff
@@ -18,16 +18,22 @@ version 1.0
 task CollectQualityYieldMetrics {
   input {
     File input_bam
+    File? input_bam_index
```

Comment:
Am I missing it or do we need need this passed in?

Reply:
line 101 in WholeGenomeSingleSampleQc.wdl?

Reply:
Or do you mean passed in to the Picard comamnd? Picard does not actually accept the BAM/CRAM index as an input. However, in order to localize data on the host, Cromwell has to be told it's an input so it retains the index with the actual BAM/CRAM.

Reply:
Got it, that makes sense!